testscommon/factory/bootstrapperFromStorageFactoryStub.go

```diff
@@ -3,19 +3,16 @@ package factory
 import (
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/process/sync/storageBootstrap"
+	"github.com/multiversx/mx-chain-go/testscommon"
 )
 
 // BootstrapperFromStorageFactoryStub -
 type BootstrapperFromStorageFactoryStub struct {
-	CreateBootstrapperFromStorageCalled func(args storageBootstrap.ArgsShardStorageBootstrapper) (process.BootstrapperFromStorage, error)
 }
 
 // CreateBootstrapperFromStorage -
-func (b *BootstrapperFromStorageFactoryStub) CreateBootstrapperFromStorage(args storageBootstrap.ArgsShardStorageBootstrapper) (process.BootstrapperFromStorage, error) {
-	if b.CreateBootstrapperFromStorageCalled != nil {
-		return b.CreateBootstrapperFromStorageCalled(args)
-	}
-	return nil, nil
+func (b *BootstrapperFromStorageFactoryStub) CreateBootstrapperFromStorage(_ storageBootstrap.ArgsShardStorageBootstrapper) (process.BootstrapperFromStorage, error) {
```

Comment:
Also duplicated with the one from `testscommon/factory/bootstrapperFromStorageFactoryMock.go`

Reply:
removed